Release 1.33.0 docs (#7212)
projectcontour/contour: changelogs/CHANGELOG-v1.33.0.md

Previously, it was based on Ubuntu and included a minimal OS with a package manager.
The distroless variant contains only the files required to run Envoy, improving security.

(#7170, @tsaarni)

## Update to Gateway API v1.3.0

Gateway API CRD compatibility has been updated to release v1.3.0.

Full release notes for Gateway API v1.3.0 can be found [here](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.3.0).

# Other Changes
- Update to Go 1.25.1. See the [Go release notes](https://go.dev/doc/devel/release#go1.25.minor) for more information. (#7211, @sunjayBhatia)
- Updates Envoy to v1.35.2. See the [Envoy release notes](https://www.envoyproxy.io/docs/envoy/v1.35.2/version_history/v1.35/v1.35) for more information about the content of the release. (#7197, @sunjayBhatia)

# Installing and Upgrading

For a fresh install of Contour, consult the [getting started documentation](https://projectcontour.io/getting-started/).

To upgrade an existing Contour installation, please consult the [upgrade documentation](https://projectcontour.io/resources/upgrading/).

# Compatible Kubernetes Versions

Contour v1.33.0 is tested against Kubernetes 1.32 through 1.34.

> # Community Thanks!
> We’re immensely grateful for all the community contributions that help make Contour even better! For this release, special thanks go out to the following contributors:

Reviewer: Could we manually modify this section this time with:

```suggestion
# Community Thanks!
We’re immensely grateful for all the community contributions that help make Contour even better! F
```
